web/src/features/charts/elements/AreaGraph.tsx

```diff
@@ -295,7 +295,7 @@ function AreaGraph({
           selectedTimeRange={selectedTimeRange}
           datetimes={datetimesWithNext}
           scaleWidth={containerWidth}
-          transform={`translate(${barWidth / 2} ${containerHeight})`}
+          transform={`translate(0 ${containerHeight})`}
```

Comment:
this ensures the ticks are not placed in the middle of the bars.

Reply:
We added this to make sure that the tick/line was placed in the middle of the bars, but if we switch to use a date range for the tooltips instead this makes more sense.